### ISSUE TITLE
Tooltip text alignment - allow different alignments

### ISSUE THREAD

**Issue by nlcwong** [patternfly/patternfly-react#2632]
Currently, the PF4 tooltip component's text is centered aligned.  Would like to request an enhancement to allow the customization of the text alignment to be either left or right as well as centered.

Use case:  It is possible the tooltip could be used to show more information of a status or data on a data table and might show multi-line data that would be easily scannable if they were either left or right aligned.

**Comment by rachael-phillips** [comment]
@nlcwong I'm just double checking that this is only for the tooltips on charting?

**Comment by nlcwong** [comment]
@rachael-phillips No, this is for all tooltips, charts or no charts.  There might be times where the tooltip text would go beyond a single line and some might feel that it would be easier to read the text if it was right aligned.  So the ask is not to change the default text alignment, but to add additional parameters that allows the designer/dev to be able to change the text alignment if they want to do so.

**Comment by rachael-phillips** [comment]
Yes, that would be great if you can open an issue just for charting updates in React. I'm going to move this issue to Core because non-charting tooltip enhancements should be considered in core.
